WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/home/WooPosHomeNavigation.kt

```diff
@@ -83,5 +90,29 @@ fun NavGraphBuilder.homeScreen(
     }
 }
 
+fun NavController.navigateToEligibilityScreen(reason: WooPosLaunchability.NonLaunchabilityReason) {
+    navigate("$ELIGIBILITY_ROUTE/${reason.name}")
+}
+
+fun NavGraphBuilder.eligibilityScreen(
+    onNavigationEvent: (WooPosNavigationEvent) -> Unit,
+) {
+    composable(
+        route = "$ELIGIBILITY_ROUTE/{$ELIGIBILITY_REASON_KEY}",
+        arguments = listOf(
+            navArgument(ELIGIBILITY_REASON_KEY) { type = NavType.StringType }
+        )
+    ) { entry ->
+        val reasonName = entry.arguments?.getString(ELIGIBILITY_REASON_KEY)
+        val reason = reasonName?.let { WooPosLaunchability.NonLaunchabilityReason.valueOf(it) }
+            ?: WooPosLaunchability.NonLaunchabilityReason.SiteSettingsUnavailable // Fallback if missing
```

Comment:
```suggestion
            ?: WooPosLaunchability.NonLaunchabilityReason.SiteSettingsUnavailable
```
The comment feels redundant